cmake/modules/AddRootBench.cmake

```diff
@@ -50,6 +50,36 @@ function(RB_ADD_GBENCHMARK benchmark)
                                               TIMEOUT "${TIMEOUT_VALUE}" LABELS "${ARG_LABEL}" RUN_SERIAL TRUE)
 endfunction(RB_ADD_GBENCHMARK)
 
+
+#----------------------------------------------------------------------------
+# function RB_ADD_PYTESTBENCHMARK(file_name)
+#
+# Dependency: pytest with pytest-benchmark and PyROOT modules
+#
+#----------------------------------------------------------------------------
+function(RB_ADD_PYTESTBENCHMARK file_name)
+  if(ROOT_pyroot_experimental_FOUND)
+    set(ROOT_ENV ROOTSYS=${ROOTSYS}
+        PATH=${ROOTSYS}/bin:$ENV{PATH}
+        LD_LIBRARY_PATH=${ROOTSYS}/lib:${ROOTSYS}/lib/${python_dir}:$ENV{LD_LIBRARY_PATH}
+        PYTHONPATH=${ROOTSYS}/lib:${ROOTSYS}/lib/${python_dir}:$ENV{PYTHONPATH})
+  elseif(ROOT_pyroot_FOUND)
+    set(ROOT_ENV ROOTSYS=${ROOTSYS}
+        PATH=${ROOTSYS}/bin:$ENV{PATH}
+        LD_LIBRARY_PATH=${ROOTSYS}/lib:$ENV{LD_LIBRARY_PATH}
+        PYTHONPATH=${ROOTSYS}/lib:$ENV{PYTHONPATH})
+  else()
+    message(STATUS "ROOT was configured without PyROOT support. Python benchmarks will be disabled!")
+  endif()
+  if(PYTEST_FOUND AND (ROOT_pyroot_experimental_FOUND OR ROOT_pyroot_FOUND))
```

Comment:
I understand `ROOT_pyroot_FOUND` is not set when building with PyROOT experimental ON?

Reply:
I thought we need to check a case when we don't have pyroot_experimental ON, but only pyroot ? What are your plans for 6.22? There will be only pyroot option?

Reply:
Well for 6.22 we will have a `python_legacy` or similar, in order to build with old PyROOT. Here I was just wondering if `ROOT_pyroot_experimental_FOUND` also implies `ROOT_pyroot_FOUND` (it might not be the case). But if it is the case, it would be enough to check `ROOT_pyroot_FOUND` here.